sql/core/src/test/scala/org/apache/spark/sql/CsvFunctionsSuite.scala

```diff
@@ -294,10 +294,19 @@ class CsvFunctionsSuite extends QueryTest with SharedSparkSession {
   }
 
   test("to_csv with option (nullValue)") {
```

Comment:
Could you explain, please, why did you change the test?

Reply:
Because `NullType` cannot be read back through `from_csv`.
In the new logic, it will throw `DATATYPE_MISMATCH.UNSUPPORTED_INPUT_TYPE` exception.
And as far as I understand, this UT is mainly to test whether `option (nullValue)` is effective,
The test `target` of changed UT are consistent with this.